### PR TITLE
Adding happychat configuration values

### PR DESCRIPTION
**Pull request by beaucollins**
happychat_url - the url the happychat client connects to
feature.happychat - wether or not the happychat client is configured

For more context this is part of a bigger Happychat feature (#7757)

Test live: https://calypso.live/?branch=add/happychat-config

### CODE REVIEW

**Comment by mjangda** [comment]
Minor: we could probably move `happychat_url` to the shared config since it's the same across environments.

:shipit:

**Comment by beaucollins** [comment]
Makes sense.

**Comment by beaucollins** [comment]
Do `*.local.json` overrides still work with values from `_shared.json`?

**Comment by beaucollins** [comment]
Yes, the local overrides still work as expected.

File: config/stage.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
 		"community-translator": true,
 		"desktop-promo": true,
 		"guided-tours": true,
+		"happychat": true,
```
Comment (mtias) [inline config/stage.json]: We usually first turn on wpcalypso / horizon before testing in staging. Enabled in development as early as possible, and then in wpcalypso as soon as it doesn't break other areas. Once it's ready, we enable in staging and do final testing.

**Pull request merged**